src/System.Net.WebProxy/tests/WebProxyTest.cs

```diff
@@ -175,6 +175,7 @@ public static IEnumerable<object[]> BypassOnLocal_MemberData()
             yield return new object[] { new Uri($"http://{IPAddress.None}"), false };
         }
 
+        [ActiveIssue(20137, TargetFrameworkMonikers.Uap)]
```

Comment:
NIT: I thought we were trying to instead of just having issue number here, pass in a string with the full url of the issue. I'm fine eitherway, we should just have a convention and try to follow it, whatever it is.